ceph.spec.in

```diff
@@ -61,6 +78,11 @@ Requires: systemd
 %if 0%{with cephfs_java}
 BuildRequires: sharutils
 %endif
+%if 0%{with selinux}
+BuildRequires:	checkpolicy
+BuildRequires:	selinux-policy-devel
+BuildRequires:	/usr/share/selinux/devel/policyhelp
```

Comment:
@branto1 i have following errror
```
make -j1 -f /usr/share/selinux/devel/Makefile ceph.pp
cat: /selinux/mls: No such file or directory
```

when `make`ing in the `selinux` directory, after installing `selinux-policy-doc`. shall we add `selinux-policy-mls` to the build-deps as well?